### PR TITLE
Release: v3.6.8

### PR DESCRIPTION
**Pull request by HoonBaek**
* Improve the index_d_ts
  * Add props types for Channel/component/FileViewer
  * Add props types for ui/FileViewer
  * Import CoreMessageType and SendableMessageType from SendbirdUIKitGlobal
  * Use destructuring import for SendbirdUIKitGlobal
* Add ChangeLog for v3.6.8
```
## [v3.6.8] (Sep 1 2023)
### Feats:
* Update `ui/FileViewer` to support multiple images
  * Modify the props structure
    ```typescript
    export enum ViewerTypes {
      SINGLE = 'SINGLE',
      MULTI = 'MULTI',
    }
    interface SenderInfo {
      profileUrl: string;
      nickname: string;
    }
    interface FileInfo {
      name: string;
      type: string;
      url: string;
    }
    interface BaseViewer {
      onClose: (e: React.MouseEvent) => void;
    }
    interface SingleFileViewer extends SenderInfo, FileInfo, BaseViewer {
      viewerType?: typeof ViewerTypes.SINGLE;
      isByMe?: boolean;
      disableDelete?: boolean;
      onDelete: (e: React.MouseEvent) => void;
    }
    interface MultiFilesViewer extends SenderInfo, BaseViewer {
      viewerType: typeof ViewerTypes.MULTI;
      fileInfoList: FileInfo[];
      currentIndex: number;
      onClickLeft: () => void;
      onClickRight: () => void;
    }
    export type FileViewerComponentProps = SingleFileViewer | MultiFilesViewer;
    ```
* Export misc. utils
  * `Channel/utils/getMessagePartsInfo`
  * `Channel/utils/compareMessagesForGrouping`
  * `Message/hooks/useDirtyGetMentions`
  * `ui/MessageInput/hooks/usePaste`
* Provide fetchChannelList method via ChannelListContext

### Fixes:
* Apply some props which is related to the `metadata` to the ChannelListQuery
  * Add metadataKey, metadataValues, and metadataStartsWith to the Channel.queries.channelListQuery
  * How to use
    ```javascript
    <Channel or ChannelProvider
      queries={{
        channelListQuery: {
          metadataKey: 'isMatching',
          metadataValues: ['true'],
        }
      }}
    />
    ```
* Improve types of `ui/FileViewer` and `Channel/component/FileViewer`
  * Add some props that have been missed
* Fix `<ImageRenderer />` not converting number to pixel string
* Modify the yypes on useChannelContext & useThreadContext
  * `useChannelContext.setQuoteMessage` should accept `UserMessage | FileMessage`
  * `useThreadContext.sendMessage` should be `string`
```

### CODE REVIEW

**Review by liamcho — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+    export type FileViewerComponentProps = SingleFileViewer | MultiFilesViewer;
+    ```
+* Export misc. utils
+  * `Channel/utils/getMessagePartsInfo`
+  * `Channel/utils/compareMessagesForGrouping`
+  * `Message/hooks/useDirtyGetMentions`
+  * `ui/MessageInput/hooks/usePaste`
+
+### Fixes:
+* Apply some props which is related to the `metadata` to the ChannelListQuery
```
Comment (AhyoungRyu) [inline CHANGELOG.md:46]: nit;

```suggestion
* Apply some props which are related to the `metadata` to the ChannelListQuery
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
+          metadataKey: 'isMatching',
+          metadataValues: ['true'],
+        }
+      }}
+    />
+    ```
+* Improve types of `ui/FileViewer` and `Channel/component/FileViewer`
+  * Add some props that have been missed
+* Fix `<ImageRenderer />` not converting number to pixel string
+* Modify the yypes on useChannelContext & useThreadContext
```
Comment (AhyoungRyu) [inline CHANGELOG.md:62]:
```suggestion
* Modify the types on useChannelContext & useThreadContext
```

**Review by AhyoungRyu — APPROVED** [review]
👍

**Pull request merged**